libs/features/mas/web-components/src/global.css.js

```diff
@@ -609,6 +609,17 @@ div[slot="footer"] {
     text-align: center;
 }
 
+sp-button a {
```

Comment:
this might affect someone in Milo who used sp-button outside of Merch team
Maybe we can change it to `sp-button a[is="checkout-link"]`?

Reply:
although no one should have sp-button a in theory and this is not elegant, but the only way to have checkout links that look like a CTA without consonant styles.